beam_sink/sinks/mysql.py

```diff
@@ -41,16 +66,8 @@ def expand(self, pcoll):
 
 
 class _Query(beam.DoFn):
-    """MySQLQuery
-    Query MySQL databases
-    Args:
-        config: Supports passing any config elements to the mysql.connector library. Most
-        commonly this will be:
-        host - host of database
-        port - 3306 is the MySQL default
-        username - User to connect as
-        password -  Password for username
-        database - The database to connect to
+    """Query.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)